src/hardware/gus.cpp

```diff
@@ -441,21 +445,25 @@ float Voice::GetSample(const ram_array_t &ram) noexcept
 	return sample;
 }
 
-AudioFrame Voice::RenderFrame(const ram_array_t &ram,
-                              const vol_scalars_array_t &vol_scalars,
-                              const pan_scalars_array_t &pan_scalars)
+void Voice::RenderVoiceToFrames(const ram_array_t& ram,
+                                const vol_scalars_array_t& vol_scalars,
+                                const pan_scalars_array_t& pan_scalars,
+                                std::vector<AudioFrame>& frames)
```

Comment:
It's in the `Voice` class, so RenderVoice is redundant. Call it `RenderFrames` or just `Render`?

Reply:
Yup.. before I was called 'RenderFrame' (in the voice class), and we also had 'RenderFrame' in the Gus class; I was worried they were confusing or ambiguous, so renamed them.. but with your comments here I think they originals were fine :-)  Will rename them back.

Reply:
Yeah I think that's better 😄 The scope is still very clear, e.g. `voice->RenderFrames(...)`.